### ISSUE TITLE
Leave additional XCode/VS projects untouched

### ISSUE THREAD

**Issue by leehu** [WeAreROLI/JUCE#258]
Please include:
Hi, as I use libraries rather than modules, when I save a projucer project I have to go into VStudio (or XCode) and re-add the external projects and re-set the dependencies.

I was wondering whether it’s possible for Projucer, before writing the new project files, to load the existing ones, take a note of any addition projects that have been added and then re-add then when writing the new project file.

Thx

**Comment by leehu** [comment]
Hi - obviously this situation is made worse as the number of platforms increases as every time an update is made to producer the same operation needs to be performed to win, osx, ios, android projects.

**Comment by julianstorer** [comment]
It's not exactly trivial to parse the existing project files. We've obviously thought about this and may have a go in the future, but it really wouldn't be simple.

**Comment by leehu** [comment]
Yes, I guessed as much, but just thought I'd throw it up there anyway... :)

**Comment by leehu** [comment]
What about a list of projects to add in Projucer - you must already have the code for creating/adding projects to the solutions that you create for opening in the dev env so you could just add these at the same time? No need to parse existing solution files then...

**Comment by leehu** [comment]
Any more thoughts on this Jules, even being able to add dependent projects in Projucer and have them appear would be such a timesaver. thx

**Comment by francoisbecker** [comment]
@leehu Why wouldn't you create an empty VS solution and add the jucer-generated projects as well as your external projects to the solution?

**Comment by leehu** [comment]
Hi, I see what you're saying. Not thought of that.

I guess that's a solution, but something I then have to handle across all platforms - having Projucer be able to specify the dependencies to add to the generated projects across all those platforms
